### PR TITLE
a2-markup-a-readme

### PR DESCRIPTION
**Pull request by valentinavargas31**

### CODE REVIEW

File: src/mm/html-css/a2-markup-a-readme/index.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```
Comment (Iregui44) [inline src/mm/html-css/a2-markup-a-readme/index.html:4]:
```suggestion
    <meta charset="UTF-8">
```

File: src/mm/html-css/a2-markup-a-readme/index.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Iregui44) [inline src/mm/html-css/a2-markup-a-readme/index.html:5]:
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

File: src/mm/html-css/a2-markup-a-readme/index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    <title>REACT.</title>
+  </head>
+  <body>
+    <h1>React.</h1>
```
Comment (Iregui44) [inline src/mm/html-css/a2-markup-a-readme/index.html:9]: Tittles with puntutation?

File: src/mm/html-css/a2-markup-a-readme/index.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+    <h2>Examples</h2>
+    <hr />
+
+    <p>
+      We have several examples
+      <a href="https://react.dev/">on the website.</a> Here is the first one to
+      get you started:
+    </p>
+
+    <p>import { createRoot } from "react-dom/client";</p>
```
Comment (Iregui44) [inline src/mm/html-css/a2-markup-a-readme/index.html:144]: Is better to use <code> tag when adding code to your pages.

File: src/mm/html-css/a2-markup-a-readme/index.html (hunk 1 above)
Comment (valentinavargas31) [reply src/mm/html-css/a2-markup-a-readme/index.html:4]: It is placed automatically due to the format configuration, but I solve it

File: src/mm/html-css/a2-markup-a-readme/index.html (hunk 2 above)
Comment (valentinavargas31) [reply src/mm/html-css/a2-markup-a-readme/index.html:5]: ok

File: src/mm/html-css/a2-markup-a-readme/index.html (hunk 3 above)
Comment (valentinavargas31) [reply src/mm/html-css/a2-markup-a-readme/index.html]: I already corrected it

File: src/mm/html-css/a2-markup-a-readme/index.html (hunk 4 above)
Comment (valentinavargas31) [reply src/mm/html-css/a2-markup-a-readme/index.html]: okey

File: src/mm/html-css/a2-markup-a-readme/index.html (hunk 1 above)
Comment (valentinavargas31) [reply src/mm/html-css/a2-markup-a-readme/index.html:4]: I tried to remove "/" but the commit usually adds it again automatically

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**